### ISSUE TITLE
different base models problem

### ISSUE THREAD

**Issue by CynthiaLai** [cardwing/Codes-for-Lane-Detection#179]
Hello, Cardwing
I want to confirm if all your code shared, different directories ( ENet-BDD100K-Torch, ENet-Label-Torch, ENet-TuSimple-Torch, ERFNet-CULane-PyTorch, SCNN-Tensorflow ) means you tried to use different backbones all contained SAD on different datasets? 
thank you!

**Comment by cardwing** [comment]
Currently, only ENet-Label-Torch contains the SAD module. However, you can easily add this module to other base models.
